journal-entry/info.json

```diff
@@ -2,8 +2,8 @@
   "name": "Journal entry",
   "identifier": "journal-entry",
   "script": "journal-entry.qml",
-  "version": "1.0.0",
-  "minAppVersion": "17.05.6",
+  "version": "1.1.0",
+  "minAppVersion": "17.12.0",
```

Comment:
please don't set `"minAppVersion": "17.12.0",`, that version isn't needed and not released yet

Reply:
Hmm ok. I did that b/c otherwise people will not be able to read the description.

Reply:
I guess that's just an issue in QON, but doesn't break the script itself. :)